src/mongoc/mongoc-config.h.in

```diff
@@ -153,5 +153,13 @@
 #  undef MONGOC_NO_AUTOMATIC_GLOBALS
 #endif
 
+/*
+ * Use system crypto profile
+ */
+#define MONGOC_ENABLE_SYSTEM_CRYPTO_PROFILE @MONGOC_ENABLE_SYSTEM_CRYPTO_PROFILE@
+
+#if MONGOC_ENABLE_SYSTEM_CRYPTO_PROFILE != 1
+#  undef MONGOC_ENABLE_SYSTEM_CRYPTO_PROFILE
```

Comment:
Sorry, I mean s/MONGOC_ENABLE_SYSTEM_CRYPTO_PROFILE/MONGOC_ENABLE_CRYPTO_SYSTEM_PROFILE/g
and move it after MONGOC_ENABLE_CRYPTO.

Thanks!